events/events-team/marketing/README.md

```diff
@@ -26,16 +31,59 @@ As Marketing Lead, you are responsible for the overall marketing communications
 - Create a communication schedule - what, when, how, who  
 - Manage online presence - website, social media strategy  
 - Potentially recruit a social media coordinator role  
-- Provide updates on social media, mailing lists, and Slack throughout pre-, during-, and post-event  
-- Assist with GitHub event repo adds and edits  
-- Work with Event Lead on a recap blog  
+- Provide updates on social media, mailing lists, and Slack throughout pre-,
+  during-, and post-event
+- Assist with GitHub event repo adds and edits
+- Work with Event Lead on a recap blog
```

Comment:
Technically this should be more broad. 

"... on any necessary blog posts (such as the schedule announcement and recap blog posts)"

Reply:
Going to forgo updating that in this PR. The guide needs more work.